src/main/java/ui/components/pickers/LabelPicker.java

```diff
@@ -31,6 +31,12 @@ private void showLabelPicker(TurboIssue issue) {
         List<TurboLabel> allLabels = ui.logic.getRepo(issue.getRepoId()).getLabels();
         // create new LabelPickerDialog
         LabelPickerDialog labelPickerDialog = new LabelPickerDialog(issue, allLabels, stage);
+
+        // Needs this since dialog's size is not available before the blocking showAndWait
+        Platform.runLater(() -> {
+            labelPickerDialog.positionDialog(stage);
```

Comment:
this causes the initial jump on OS X, but without it the dialog is positioned correctly on launch.